float32[4] position_rads # pos in rads, converted in subscriber.py to encoder counts
int32[4] speed 
int32[4] accel_deccel 
bool[2] hall_vals # indicates whether the hall effect sensors are triggered